{
  "description": "Account staking reward payouts api call with all applicable params *should* work, but throws an internal error",
  "setup": {
    "entities": [
      {
        "num": 1001
      },
      {
        "num": 800
      }
    ],
    "stakingRewardTransfers": [
      {
        "accountId": "0.0.1001",
        "amount": 10,
        "consensus_timestamp": "123456789000011111",
        "payerAccountId": "0.0.800"
      },
      {
        "accountId": "0.0.1001",
        "amount": 12,
        "consensus_timestamp": "123456789000022222",
        "payerAccountId": "0.0.800"
      },
      {
        "accountId": "0.0.1001",
        "amount": 13,
        "consensus_timestamp": "123456789000033333",
        "payerAccountId": "0.0.800"
      }
    ]
  },
  "url": "/api/v1/accounts/0.0.1001/rewards?timestamp=gte:0&order=asc&limit=2",
  "url": "/api/v1/accounts/0.0.1001/rewards?timestamp=gte:0&limit=2",
  "url": "/api/v1/accounts/0.0.1001/rewards?timestamp=gte:0&order=asc",
  "url": "/api/v1/accounts/0.0.1001/rewards?order=asc&limit=2",
  "responseStatus": 500,
  "responseJson": {
    "_status": {
      "messages": [
        {
          "message": "Internal error"
        }
      ]
    }
  }
}
